external/fv3fit/fv3fit/keras/_models/shared/clip.py

```diff
@@ -20,21 +19,12 @@ class ClipConfig(PackerConfig):
             dimension to clip.
     """
 
-    clip: ClipDims = dataclasses.field(default_factory=dict)
```

Comment:
Can you update the docstring above as well? (it says it is 'usually the last dim')

Reply:
Updated.